File: AndroisLifeCycleDemo/app/src/main/java/com/example/androislifecycledemo/MainActivity.java
package com.example.androislifecycledemo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
Button button;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button=findViewById(R.id.button);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(MainActivity.this,SecondActivity.class);
                startActivity(intent);
            }
        });

    }

//    @Override
//    protected void onStop() {
//        super.onStop();
//        Toast.makeText(MainActivity.this,"onstop function called",Toast.LENGTH_SHORT).show();
//    }
    //    @Override
//    protected void onPause() {
//        super.onPause();
//        Toast.makeText(MainActivity.this,"pause function called",Toast.LENGTH_SHORT).show();
//    }
//
//    @Override
//    protected void onStart() {
//        super.onStart();
//        Toast.makeText(MainActivity.this,"Welcome back to your own App",Toast.LENGTH_SHORT).show();
//    }
//
//    @Override
//    protected void onResume() {
//        super.onResume();
//        Toast.makeText(MainActivity.this,"Explore More Activity is running",Toast.LENGTH_SHORT).show();
//    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        Toast.makeText(MainActivity.this,"App closed",Toast.LENGTH_SHORT).show();
    }
}

File: FloatingButtonApp/app/src/main/java/com/example/floatingbuttonapp/MainActivity.kt
package com.example.floatingbuttonapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import com.google.android.material.floatingactionbutton.FloatingActionButton

class MainActivity : AppCompatActivity() {
   lateinit var floatingActionButton : FloatingActionButton
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        floatingActionButton= findViewById(R.id.floating_action_button)
        floatingActionButton.setOnClickListener(View.OnClickListener {
            Toast.makeText(applicationContext,"this floating Action button",Toast.LENGTH_SHORT).show()
        })

    }
}